moped-editor/src/views/projects/projectView/ProjectMilestones/RelatedPhaseTextField.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { TextField } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import { useGridApiContext } from "@mui/x-data-grid-pro";
+
+const useStyles = makeStyles((theme) => ({
+  readOnlyInput: {
+    width: "90%",
+    paddingTop: "inherit",
+    paddingLeft: theme.spacing(1),
+  },
+}));
+
+/**
+ * @param {Integer} id - Data Grid row id
+ * @param {String} value - field value
+ * @param {String} field - name of field
+ * @param {Boolean} hasFocus - does this field have focus
+ * @param {Object} phaseNameLookupData - maps phase id to phase name
+ * @param {Boolean} usingShiftKey - is the shift key being pressed during tabbing
+ * @return {JSX.Element}
+ */
+const RelatedPhaseTextField = ({
+  value,
+  id,
+  field,
+  hasFocus,
+  phaseNameLookupData,
+  usingShiftKey,
+}) => {
+  const ref = React.useRef(null);
+  const apiRef = useGridApiContext();
+  const classes = useStyles();
+
+  // Because this field not editable, it cannot be focused and if a user is tabbing across the cells
+  // the focus should be forwarded to the next one in the row
+  React.useEffect(() => {
+    if (hasFocus) {
+      if (usingShiftKey) {
```

Comment:
you could add a comment somewhere that usingShiftKey is for checking if a user is trying to tab backwards, i was confused about the purpose of this for awhile

Reply:
ill add a comment b/c I also dont tab very often